<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowContrat</class>
 <widget class="QMainWindow" name="MainWindowContrat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindowContrat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0105263, y1:0.051, x2:1, y2:1, stop:0 rgba(107, 154, 255, 255), stop:1 rgba(222, 148, 222, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>741</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>supprimer un Contrat</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
      </property>
      <property name="text">
       <string>code a supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="contrat_le_id_sup">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="supp">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MODIFIER">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>320</y>
        <width>93</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>70</y>
        <width>351</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Code</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_codemod"/>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>num</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_newnum"/>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>location</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_newloc"/>
       </item>
      </layout>
      <zorder>label_11</zorder>
      <zorder>lineEdit_newnum</zorder>
      <zorder>label_13</zorder>
      <zorder>lineEdit_newloc</zorder>
      <zorder>label_12</zorder>
      <zorder>lineEdit_codemod</zorder>
     </widget>
     <widget class="QLineEdit" name="contrat_values">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>280</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>the statistics</string>
      </property>
     </widget>
     <widget class="QPushButton" name="statistics">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>310</y>
        <width>93</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
      </property>
      <property name="text">
       <string>statistics</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="contrat_domdom">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>250</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>250</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>domain</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exportpdf">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
      </property>
      <property name="text">
       <string>export pdf</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exportpng">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>10</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
      </property>
      <property name="text">
       <string>export un contrat</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="contrat_exportid">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="contrat_pass">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>370</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>350</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>password</string>
      </property>
     </widget>
     <zorder>label_9</zorder>
     <zorder>contrat_le_id_sup</zorder>
     <zorder>supp</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>contrat_values</zorder>
     <zorder>label_14</zorder>
     <zorder>statistics</zorder>
     <zorder>contrat_domdom</zorder>
     <zorder>label_15</zorder>
     <zorder>exportpdf</zorder>
     <zorder>exportpng</zorder>
     <zorder>contrat_exportid</zorder>
     <zorder>label_17</zorder>
     <zorder>MODIFIER</zorder>
     <zorder>contrat_pass</zorder>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>stistic</string>
     </attribute>
     <widget class="QFrame" name="horizontalFrame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-20</y>
        <width>661</width>
        <height>451</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
     <widget class="QLabel" name="percent">
      <property name="geometry">
       <rect>
        <x>445</x>
        <y>60</y>
        <width>271</width>
        <height>111</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="percent_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>161</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>afficher un Contrat</string>
     </attribute>
     <widget class="QTableView" name="tab_contrat">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>711</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>ajouter un Contrat</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>711</width>
        <height>411</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajoute</string>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>40</y>
         <width>141</width>
         <height>261</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLineEdit" name="contrat_lineEdit_d"/>
        </item>
        <item>
         <widget class="QLineEdit" name="contrat_lineEdit_nom"/>
        </item>
        <item>
         <widget class="QLineEdit" name="contrat_lineEdit_loc"/>
        </item>
        <item>
         <widget class="QLineEdit" name="contrat_lineEdit_email"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>340</y>
         <width>93</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>54</y>
         <width>61</width>
         <height>241</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
          </property>
          <property name="text">
           <string>domaine</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
          </property>
          <property name="text">
           <string>nom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
          </property>
          <property name="text">
           <string>location</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
          </property>
          <property name="text">
           <string>password</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>92</width>
         <height>251</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
          </property>
          <property name="text">
           <string>Code</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
          </property>
          <property name="text">
           <string>num</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
          </property>
          <property name="text">
           <string>prix</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="styleSheet">
           <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
          </property>
          <property name="text">
           <string>temps de debut</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
          </property>
          <property name="text">
           <string>temps de fin</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_5">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>141</width>
         <height>261</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="contrat_lineEdit_code"/>
        </item>
        <item>
         <widget class="QLineEdit" name="contrat_lineEdit_num"/>
        </item>
        <item>
         <widget class="QLineEdit" name="contrat_lineEdit_prix"/>
        </item>
        <item>
         <widget class="QLineEdit" name="contrat_lineEdit_td"/>
        </item>
        <item>
         <widget class="QLineEdit" name="contrat_lineEdit_tf"/>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="afficheur">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>340</y>
         <width>93</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:20px;
background-color: rgb(62, 125, 188);</string>
       </property>
       <property name="text">
        <string>afficher</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>190</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>jj/mm/AAAA</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>551</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
